apps/ui/components/reservation/Step0.tsx

```diff
@@ -78,7 +78,7 @@ const Step0 = ({
 
   const {
     watch,
-    formState: { errors, isSubmitted },
+    formState: { errors, isSubmitted, isValid },
```

Comment:
Does `isSubmitting` work here instead of `isSubmitted`? Would fix issues with mutation failing and showing the loader still?